Vagrantfile.example

```diff
@@ -16,14 +16,16 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
      config.hostmanager.manage_host = true
  end
 
- # install vagrant-cachier for some caching goodness
- if Vagrant.has_plugin?("vagrant-cachier")
-    config.cache.scope = :box
-    config.cache.synced_folder_opts = {
-        type: :nfs,
-        mount_options: ['rw', 'vers=4', 'tcp']
-    }
- end
+ # vagrant plugin install vagrant-cachier for some caching goodness
+ # Currently broken due to cachier making yum-based assumptions
+ #  https://github.com/fgrehm/vagrant-cachier/issues/163
+ # if Vagrant.has_plugin?("vagrant-cachier")
+ #    config.cache.scope = :box
+ #    config.cache.synced_folder_opts = {
+ #        type: :nfs,
+ #        mount_options: ['rw', 'vers=4', 'tcp']
+ #    }
+ # end
```

Comment:
I'd normally do just that, but this is the example file. I like that it includes an example of conditionally using vagrant-cachier. I can fix the comment, at least, to be more clear about what cachier actually caches, and otherwise have it serve as a better example.